### PR TITLE
feat: added t function of paddle frontend and related test cases

### LINKED ISSUE #22865: t

### PR DESCRIPTION
**Pull request by sachelsout**
# PR Description 

This PR solves the issue named 't' (which comes under paddle.tensor.linalg). The function definition and related test cases done. All the test cases are passing locally.

## Related Issue 

Close #22865 

## Checklist 

- [x] Did you add a function?
- [x] Did you add the tests?
- [x] Did you follow the steps we provided?

### Socials: 

<a href="https://twitter.com/rohan_dawkhar" target="blank">rohan_dawkhar</a>

### CODE REVIEW

**Comment by sachelsout** [comment]
Hi @Infrared1029 , could you please review my PR 😃

**Comment by ivy-leaves** [comment]
If you are working on an open task, please **edit** the PR description to link to the issue you've created.

For more information, please check [ToDo List Issues Guide](https://unify.ai/docs/ivy/overview/contributing/the_basics.html#todo-list-issues).

Thank you :hugs:

**Comment by sachelsout** [comment]
Hi @Infrared1029 , I resolved merge conflicts and tested the function locally again. But one of the test case is failing (keyError: 'CPU'). This error popped up after I resolved the merge conflicts. Attaching below the screenshot of the error.

image

File: ivy/functional/frontends/paddle/tensor/linalg.py [hunk 1]
```diff
@@ -1,2 +1,220 @@
...
+    elif len(ivy.shape(input)) == 2:
+        perm = [1, 0]
+        return ivy.permute_dims(input, axes=perm)
+
+
+# transpose
+@with_unsupported_dtypes({"2.5.1 and below": ("uint8", "int8", "int16")}, "paddle")
+@to_ivy_arrays_and_back
+def transpose(x, perm, name=None):
+    return ivy.permute_dims(x, axes=perm)
```
Comment (Sarvesh-Kesharwani) [inline ivy/functional/frontends/paddle/tensor/linalg.py:220]: Remove these changes as well.

**Review by Sarvesh-Kesharwani — CHANGES_REQUESTED** [review]

**Comment by ivy-seed** [comment]
This PR has been labelled as stale because it has been inactive for more than 7 days. If you would like to continue working on this PR, then please add another comment or this PR will be closed in 7 days.

**Comment by ivy-seed** [comment]
This PR has been closed because it has been marked as stale for more than 7 days with no activity.

**Pull request closed without merging**